test/tests/open-test.js

```diff
@@ -35,7 +35,8 @@ describe('open', function () {
                 throw new Error('Promise rejection expected');
             })
             .catch(function (err) {
-                expect(err.message).eql(messages.getText(messages.MESSAGES.unableToRunBrowser, browserInfo.path));
+                expect(err.message).eql('Unable to run the browser. ' +
```

Comment:
Here you need to split lines, but try to achieve equal lines length for better readability.